irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -21,10 +21,9 @@
 #include "ametsuchi/storage.hpp"
 
 #include <cmath>
-#include <shared_mutex>
-
 #include <nonstd/optional.hpp>
 #include <pqxx/pqxx>
+#include <shared_mutex>
```

Comment:
what is the purpose to add mutex into the header?

Reply:
I think for rw_lock_

Reply:
So, how it works before?

Reply:
How it works before?

Reply:
I just reordered includes with clang-format. I suppose it is used for lock on write.